Podcast/SubscriptionCollectionViewCell.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  SubscriptionCollectionViewCell.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 3/6/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class SubscriptionCollectionViewCell: RecommendedSeriesCollectionViewCell {
```

Comment:
why not just standardize it by only having SeriesGridCollectionViewCell class that we can use everywhere we have those series cell blocks.